src/map_preview_screen.c

```diff
@@ -525,33 +552,33 @@ static void sub_80F83D0(u8 taskId)
     }
 }
 
-const struct MapPreviewScreen * sub_80F8544(u8 id)
+const struct MapPreviewScreen * sub_80F8544(u8 mapsec)
 {
     u8 idx;
 
-    idx = sub_80F80E4(id);
+    idx = sub_80F80E4(mapsec);
     if (idx == MPS_COUNT)
     {
         return NULL;
     }
     else
     {
-        return &gUnknown_843E9E8[idx];
+        return &sMapPreviewScreenData[idx];
     }
 }
 
-u16 sub_80F856C(u8 id)
+u16 sub_80F856C(u8 mapsec)
 {
     u8 idx;
     u16 flagId;
 
-    idx = sub_80F80E4(id);
+    idx = sub_80F80E4(mapsec);
     if (idx == MPS_COUNT)
     {
         return 0;
     }
-    flagId = gUnknown_843E9E8[idx].flagId;
-    if (gUnknown_843E9E8[idx].forceFirstTime == 0)
+    flagId = sMapPreviewScreenData[idx].flagId;
+    if (sMapPreviewScreenData[idx].forceFirstTime == 0)
```

Comment:
FALSE

Reply:
RE sub_80F8154: If you're referring to `u8 forceFirstTime`, a value of 2 is expected for this argument so the type is correct as is.